src/test/java/io/strimzi/kafka/bridge/http/ConsumerGeneratedNameTest.java

```diff
@@ -54,19 +58,21 @@ void createConsumerNameNotSet(VertxTestContext context) throws InterruptedExcept
                 .as(BodyCodec.jsonObject())
                 .sendJsonObject(json, ar -> {
                     context.verify(() -> {
+                        LOGGER.info("Verifying that consumer name is created with random suffix");
```

Comment:
at this point this test should be something like `createConsumerNameAndBridgeIdNotSet` to check that it starts with "kafka-bridge-consumer-" and it has a random suffix. Maybe we should havea  test where the consumer name is not set but the `bridge.id` is and checking that the consumer name is the `bridge.id` value plus a random suffix.

Reply:
I agree with you, can you please open an issue for that? I will then work on that.

Reply:
Here it is #404 (funny issue code!)

Reply:
Done I have also added a new test case which you described.